crates/turbopath/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "turbopath"
+version = "0.1.0"
+edition = "2021"
+license = "MPL-2.0"
```

Comment:
The diff from this PR (ported from everybody else) has rounds-to-zero original code and/or the implementation is entirely constrained by the API such that there was actually no choice in implementation.

MPL-2.0 is the preferred license of the Turboverse for our own work and it is arguable as to where to draw the line on this one for "owned work." So I punted: first commit will be Apache/MIT. Further work on this will begin to introduce clearly owned code, and should be licensed under MPL-2.0.